netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -182,7 +193,9 @@ WriteQueue getWriteQueue() {
    */
   void returnProcessedBytes(Http2Stream stream, int bytes) {
     try {
-      decoder().flowController().consumeBytes(ctx, stream, bytes);
+      decoder().flowController().consumeBytes(stream, bytes);
+      decoder().flowController().writeWindowUpdates(ctx);
+      ctx.flush();
```

Comment:
This looks like a flush we will need to remove later. I'm not worried about it being in this PR, but am I right that we will be trying to remove it?